docs/release_guidelines.md

```diff
@@ -0,0 +1,23 @@
+# OCS-CI Releases
+
+OCS-CI releases are generated in conjunction with OCS releases, A release is cut
+as soon as OCS is Generally Available and available in open market place.
+
+The release helps to checkout specific versions of tests that were run during
+the release and serves as baseline for the specific version of OCS.
+
+# Release Process
+1. Update the setup.py to match the ocs version and commit the change.
+2. Create a tag specific to commit that matches the latest ocs-ci that was used to run
```

Comment:
I think that anyway we need to tag the commit created above which will match the version. If we will tag the commit which we ran the regression cycle with then we will never have the proper version in the setup.py for this release.  I think that we should just do that release the same date as we do the release of OCS and we don't have to be that strict to have exact commit we ran the regression cycle with. WDYT?

So if that OK I would suggest to create the tag on the commit created in 1st step here @vasukulkarni

Reply:
If we do the same day release I think we are safe, but in case we miss we still have in this document that works. The only thing the user has to skip the same day is to not provide commit id in that case.

Reply:
We can make it as part of our release process and just as we do live testing we will do the release of OCS-CI as well.  @ebenahar @RazTamir  @dahorak @jimdevops19 @clacroix12 @shylesh @vavuthu   FYI.  WDYT?

Reply:
@petr-balogh we can continue to have that discussion, but in the mean time lets update the version and get this one merged if everything is looking fine?   soon there will be one more update to setup.py

Reply:
@petr-balogh Agree.